modules/ai/assets/js/editor/pages/form-layout/index.js

```diff
@@ -254,7 +262,15 @@ const FormLayout = ( {
 							onCancel={ () => setShowUnsavedChangesAlert( false ) }
 						/>
 					) }
-
+					{ shouldRenderWebApp && (
+						<AttachDialog
+							type={ ATTACHMENT_TYPE_URL }
+							url={ promptInputRef.current.value }
+							onAttach={ () => {} }
```

Comment:
It should preview the attachment, etc.

```suggestion
							onAttach={ onAttach }
```